src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -86,30 +113,36 @@ var fluid_2_0 = fluid_2_0 || {};
                 func: "{that}.applier.requestChange",
                 args: ["isShowing", true]
             },
+            setAriaStates: {
+                funcName: "fluid.slidingPanel.setAriaStates",
+                args: ["{that}", "{that}.model.isShowing"]
+            },
             togglePanel: {
                 funcName: "fluid.slidingPanel.togglePanel",
-                args: ["{that}"]
+                args: ["{that}", "{that}.model.isShowing"]
             },
             refreshView: {
                 funcName: "fluid.slidingPanel.refreshView",
                 args: ["{that}"]
             }
         },
-        model: {
-            isShowing: false
-        },
         animationDurations: {
             hide: 400,
             show: 400
         }
     });
 
-    fluid.slidingPanel.togglePanel = function (that) {
-        that.applier.requestChange("isShowing", !that.model.isShowing);
+    fluid.slidingPanel.togglePanel = function (that, isShowing) {
+        that.applier.requestChange("isShowing", !isShowing);
```

Comment:
I think I made this change when I was writing the setAriaStates function, but I don't think it's necessary now that I look at it again